composer.json

```diff
@@ -1,14 +1,117 @@
 {
     "name": "zendframework/skeleton-application",
-    "description": "Skeleton Application for ZF2",
+    "description": "Skeleton Application for Zend Framework",
     "license": "BSD-3-Clause",
     "keywords": [
         "framework",
         "zf2"
     ],
     "homepage": "http://framework.zend.com/",
+    "minimum-stability": "dev",
```

Comment:
Need it for now if you want to use zend-test; without this set, that component will not install (and it needs to be pinned to dev-develop until zend-mvc v3 is released, which is when we can do a zend-test v3 release).